ggshield/cmd/iac/scan/diff.py

```diff
@@ -57,57 +63,103 @@ def scan_diff_cmd(
     display_warning(
         "This feature is still in beta, its behavior may change in future versions."
     )
+
     if directory is None:
         directory = Path().resolve()
     update_context(ctx, exit_zero, minimum_severity, ignore_policies, ignore_paths)
+
     result = iac_scan_diff(ctx, directory, ref, staged)
     return display_iac_scan_diff_result(ctx, directory, result)
 
 
 def iac_scan_diff(
-    ctx: click.Context, directory: Path, ref: str, include_staged: bool
+    ctx: click.Context,
+    directory: Path,
+    previous_ref: Optional[str],
+    include_staged: bool,
+    current_ref: Optional[str] = None,
 ) -> Union[IaCDiffScanResult, IaCSkipScanResult, None]:
+    """
+    Performs a diff scan for IaC vulnerabilities,
+    comparing two git reference. Vulnerabilities are flagged as new, removed or
+    remaining depending on whether they appear in the `current_ref` and `previous_ref`
+    git references.
+
+    :param ctx: click.Context with CLI arguments
+    :param directory: path to the location we want to scan. Defaults to the current
+    directory.
```

Comment:
There is no default for the directory here.

Reply:
removed